domain-models-interface-extensions/src/main/java/org/folio/rest/tools/AnnotationGrabber.java

```diff
@@ -289,6 +288,24 @@ public static JsonObject getParameterNames(Method method) throws Exception {
     return retObject;
   }
 
+  private static boolean isPossibleHttpMethod(String method) {
+    switch (method) {
+    case "javax.ws.rs.PUT":
+    case "javax.ws.rs.POST":
+    case "javax.ws.rs.DELETE":
+    case "javax.ws.rs.GET":
+    case "javax.ws.rs.OPTIONS":
+    case "javax.ws.rs.HEAD":
+    case "javax.ws.rs.TRACE":
+    case "javax.ws.rs.CONNECT":
+    case "org.folio.rest.jaxrs.resource.support.OPTIONS":
```

Comment:
I'm not familiar enough with this code to know if this was done intentionally or not, but OPTIONS is here twice...

Is this needed, or a mistake?

Reply:
The new one is needed.
I don't know whether the existing one from javax.ws.rs is need, might be in use in older versions. To avoid a breaking change we keep it.